auacm/app/static/js/controllers/ScoreboardController.js

```diff
@@ -25,9 +25,16 @@ app.controller('ScoreboardController', ['$scope', '$http', '$routeParams',
             var solved = 0;
             var time = 0;
             for (var problemName in team.problemData) {
+
                 if (team.problemData[problemName].status === 'correct') {
                     solved++;
-                    time += team.problemData[problemName].problemTime;
+                    team.problemData[problemName].penaltyTime =
+                        (team.problemData[problemName].submitCount - 1) * 20;
+                    time += team.problemData[problemName].submitTime +
+                        team.problemData[problemName].penaltyTime;
+                } else {
+                    team.problemData[problemName].penaltyTime =
+                        (team.problemData[problemName].submitCount) * 20;
```

Comment:
I told you wrong the first time; my sincerest apologies. These (lines 32, 34, and 37) need to be indented an additional 4 spaces. Google style says 4, but they use 2 space indents where we use 4, so that needs to be doubled.